wormhole-connect/src/components/Modal.tsx

```diff
@@ -61,10 +61,16 @@ function Modal({ open, width, closable, children, onClose }: Props) {
   return (
     <Dialog
       open={open}
-      onClose={onClose}
+      onClose={(event, reason) => {
```

Comment:
Might want to consider use callback to avoid re-render / re-attaching the event every render.

Reply:
This event never seems to fire for me (after removing the onClick)